src/main/scala/beam/agentsim/agents/household/HouseholdActor.scala

```diff
@@ -135,10 +147,67 @@ object HouseholdActor {
     implicit val pop: org.matsim.api.core.v01.population.Population = population
 
     private var availableVehicles: List[BeamVehicle] = Nil
+    private var cavPlans: List[CAVSchedule] = List()
```

Comment:
Can we use a `mutable.List` instead of a mutable reference to an immutable List. This is the worst case for performance. Same for all the other. In fact, I would refactor the existing `availableVehicles` to a `mutable.List` also